bundles/binding/org.openhab.binding.isy/src/main/java/org/openhab/binding/isy/ISYActiveBinding.java

```diff
@@ -0,0 +1,718 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.isy;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+import java.math.BigDecimal;
+
+import org.openhab.binding.isy.internal.ISYNodeType;
+import org.openhab.binding.isy.internal.ErrorHandler;
+import org.openhab.binding.isy.internal.ISYControl;
+import org.openhab.binding.isy.internal.ISYModelChangeListener;
+import org.openhab.binding.isy.internal.InsteonClient;
+import org.openhab.core.binding.AbstractActiveBinding;
+import org.openhab.core.library.types.DecimalType;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.library.types.OpenClosedType;
+import org.openhab.core.library.types.StringType;
+import org.openhab.core.library.types.PercentType;
+import org.openhab.core.library.types.IncreaseDecreaseType;
+import org.openhab.core.library.types.DateTimeType;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.universaldevices.device.model.UDControl;
+import com.universaldevices.device.model.UDGroup;
+import com.universaldevices.device.model.UDNode;
+import com.universaldevices.resources.errormessages.Errors;
+
+/**
+ * Implement this class if you are going create an actively polling service like
+ * querying a Website/Device.
+ * 
+ * @author Tim Diekmann
+ * @since 1.7.0
+ */
+public class ISYActiveBinding extends AbstractActiveBinding<ISYBindingProvider>
+		implements ISYModelChangeListener {
+
+	private final Logger logger = LoggerFactory
+			.getLogger(ISYActiveBinding.class);
```

Comment:
Please use the code formatting rules that [the Eclipse IDE](https://github.com/openhab/openhab-distro/blob/master/docs/sources/development/ide.md) performs on source files.